NomaiVR/Tools/HoldSignalscope.cs

```diff
@@ -156,7 +156,9 @@ private void UpdateSignalscipeZoom()
                 if (OWInput.IsNewlyPressed(InputLibrary.scopeView, InputMode.All) && ToolHelper.Swapper.IsInToolMode(ToolMode.SignalScope, ToolGroup.Suit))
                 {
                     _lens.gameObject.SetActive(!_lens.gameObject.activeSelf);
-                    _owLensCamera.SetEnabled(_lens.gameObject.activeSelf);
+
+                    if(_owLensCamera.gameObject != null)
```

Comment:
So it's possible for a component to still exist while its gameObject is null?

Reply:
Yes, I don't know why but suiting up is a way for it to happen

Reply:
Strange, I think it's only supposed to be null after it's destroyed. But if it works, sure.